project/EnsimeBuild.scala

```diff
@@ -234,7 +231,6 @@ object EnsimeBuild extends Build {
       aggregate in assembly := false,
       assemblyMergeStrategy in assembly := {
         case PathList("META-INF", "namedservices", xs @ _*) => MergeStrategy.filterDistinctLines
```

Comment:
a bunch of this other stuff is probably netbeans to. If you remove it, and `sbt ensime/assembly` still works, then it is ok to remove it.